File: Android_AMS/src/main/java/com/lay/android_ams/MainActivity.java
package com.lay.android_ams;

import android.content.Intent;
import android.os.Bundle;
import android.widget.LinearLayout;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        init();
        initView();
    }

    public void init() {
        findViewById(R.id.tvClick).setOnClickListener((v) -> {
            startActivity(new Intent(this, TargetActivity.class));
        });
    }

    public void initView() {
        LinearLayout container = findViewById(R.id.ll_loaner_container);

        for (int i = 0; i < 10; i++) {

            MyView myView = new MyView(this);
            myView.setText("借款号: " + i);

            container.addView(myView);
        }

    }
}

File: Open_EventBus/src/main/java/com/lay/open_eventbus/MainActivity.kt
package com.lay.open_eventbus

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import org.greenrobot.eventbus.EventBus
import org.greenrobot.eventbus.Subscribe
import org.greenrobot.eventbus.ThreadMode
import org.jetbrains.anko.toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        EventBus.getDefault().register(this)

        findViewById<Button>(R.id.btn).setOnClickListener {
//            EventBus.getDefault().post("str")
        }
    }

    @Subscribe(threadMode = ThreadMode.MAIN)
    fun getM(str: String) {
        toast(str)
        findViewById<Button>(R.id.btn).text = str
    }

    override fun onDestroy() {
        super.onDestroy()
//        EventBus.getDefault().unregister(this)
    }

}